qiskit/providers/models/pulsedefaults.py

```diff
@@ -140,13 +140,4 @@ class PulseDefaults(BaseModel):
         discriminator (Discriminator): Default discriminator.
     """
 
-    def __init__(self, qubit_freq_est, meas_freq_est, buffer, pulse_library,
```

Comment:
Also for consistency with the rest of the codebase - if some of the parameters end up being `required=True`, can you make sure they are included in the constructor parameters and set inside it?